packages/edit-site/src/components/global-styles/font-library-modal/resolvers.js

```diff
@@ -72,11 +72,3 @@ export async function fetchFontCollections() {
 	};
 	return apiFetch( config );
 }
-
-export async function fetchFontCollection( id ) {
-	const config = {
-		path: `/wp/v2/font-collections/${ id }`,
-		method: 'GET',
-	};
-	return apiFetch( config );
-}
```

Comment:
Why is this function being removed?

Reply:
Ohh I found why in this comment: 

> NOTE: While working on this, I noticed that the calls to load individual Font Collections were unnecessary (that data is loaded when all of the font collections are loaded) and have been removed.

That should not be like that because the initial `/font-collections/` request would be unnecessarily heavy.